bootfw/depthcharge/prepare_parts

```diff
@@ -16,9 +16,18 @@ if [ -n "$2" ]; then # if $2 is set, it means that user wants a file
 	ROOTPART=${DEVICE}p3
 else
 	DEVICE=$1
-	KERNPART=${DEVICE}1
-	KERNPART_B=${DEVICE}2
-	ROOTPART=${DEVICE}3
+	
+	# resolves issue with mmcblk devices
+	if [[ $DEVICE == mmcblk* ]]; then
```

Comment:
Maybe the check should be
`if [ -e ${DEVICE}1 ]`
to account for devices that are not mmcblk* and do use p to separate whole drive from single partitions

Reply:
@Maccraft123 not sure I follow but I have an updated commit where I switched to the single test expression (before I saw this).

 I tested this. The logic is similar, we test for mmcblk devices and set the partitions accordingly. Otherwise we use the same values you used previous.